internal/js/modules/k6/browser/common/js/injected_script.js

```diff
@@ -14,6 +14,31 @@
  * limitations under the License.
  */
 
+// k6BrowserNative allows accessing native browser objects
+// even if the page under test has overridden them.
+const k6BrowserNative = (() => {
+  const iframe = document.createElement('iframe');
+  // hide it offscreen with zero size
+  iframe.style.position = 'absolute';
+  iframe.style.width = '0';
+  iframe.style.height = '0';
+  iframe.style.border = '0';
+  iframe.style.top = '-9999px';
+  iframe.style.left = '-9999px';
+  iframe.style.display = 'none';
+
+  // grab the native browser object
+  document.documentElement.appendChild(iframe);
+  const win = iframe.contentWindow;
+  document.documentElement.removeChild(iframe);
+
+  return {
+    Set: win.Set,
+    Map: win.Map,
+    // Add other native browser objects as needed.
+  }
+})();
+
```

Comment:
Not sure why this seems like a new code addition, as it was from a previous PR?

Reply:
I had to move it up in the file so that the injected script code from Playwright could work with the `k6BrowserNative` `set` and `map`.